lib/aftership/v4/configuration.rb

```diff
@@ -1,19 +1,24 @@
 module AfterShip
   module V4
     class Configuration
-      attr_accessor :api_endpoint, :api_key
-      attr_writer :timeout
+      attr_writer :connect_timeout, :send_timeout, :receive_timeout
 
       def initialize
         @api_endpoint = 'https://api.aftership.com'
-        @timeout = 3
         @headers = { 'Content-Type' => 'application/json' }
+        @connect_timeout = @send_timeout = @receive_timeout = 500 # milliseconds
```

Comment:
Is it ok to use the same number for all 3 timeouts ?
I think the receive timeout should be relatively bigger than the connect and send, what do you think ?

Reply:
I don't believe so. We can copy [the default values](https://github.com/nahi/httpclient/blob/e13cd048e5d0c94bbc1556815081c4cc87b77400/lib/httpclient/session.rb#L145-L151) from `HTTPClient`, WDYT?

Reply:
let's keep the same ratio, so 500 and 1000 ?

Reply:
Sure, let's do that!